### ISSUE TITLE
Ordered Movie Library

### ISSUE THREAD

**Issue by crazydevman** [crazydevman/mute-profanity#19]
When navigating to the Movie Library (Mute Profanity>Edit Movies) it kinda has a strange order if you've added movies to your library. So it looks like this:
Movies A-Z - Previously added movies (before installing Mute Profanity)
Movies added after Mute Profanity was installed/used

I don't understand why my newly added movies are showing up at the bottom rather than their alphabetical location? Not a big deal, but something to look at.

**Comment by FrancoisWiid** [comment]
If you are in the add-on and tap left to open the 'context' view, the search can search movies but takes you to the movie in the library instead of the movie in the add-on. Searching would ease the ordering problem and make it easier to find and run the add-on on a movie.

**Comment by crazydevman** [comment]
Good idea @FrancoisWiid, I can look into adding a search feature into the plugin
